Content.Server/Shadowkin/ShadowkinSystem.cs

```diff
@@ -166,16 +118,13 @@ private void OnRejuvenate(EntityUid uid, ShadowkinComponent component, Rejuvenat
         }
 
         EnsureComp<PsionicComponent>(uid, out var magic);
-        magic.Mana = 250;
-        magic.MaxMana = 250;
+        magic.Mana = 200;
+        magic.MaxMana = 200;
         magic.ManaGain = 0.25f;
-        magic.BypassManaCheck = true;
-        magic.Removable = false;
         magic.MindbreakingFeedback = "shadowkin-blackeye";
+        magic.NoMana = "shadowkin-tired";
```

Comment:
I think you can do this with something like this
```cs
if (!HasComp<PsionicComponent>(uid))
	AddComp(uid, new PsionicComponent
		{
			Mana = 200;
			MaxMana = 200;
			// etc...
		});
```

Reply:
Oh Neat